client/extensions/woocommerce/index.js

```diff
@@ -194,11 +178,11 @@ function getAnalyticsPath( path, params ) {
 	}
 
 	if ( '/store/settings/email/:site/:setup?' === path ) {
-		return !! params.setup ? '/store/settings/email/:site/:setup' : '/store/settings/email/:site';
+		return params.setup ? '/store/settings/email/:site/:setup' : '/store/settings/email/:site';
```

Comment:
FYI, this was changed by eslint with `181:10  error  Redundant double negation  no-extra-boolean-cast` error.